apps/backend/alembic/versions/66cc68f4e112_fix_configuration_snapshots_id_column_.py

```
@@ -0,0 +1,67 @@
+"""Fix configuration_snapshots ID column from bigint to UUID
+
+Revision ID: 66cc68f4e112
+Revises: af4ba97c1822
+Create Date: 2025-08-07 02:28:20.575166
+
+"""
+from typing import Sequence, Union
```

Ruff (0.12.2)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)